ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -981,6 +981,7 @@ private class JdbcLedgerDao(
           }.get
 
         case PersistenceEntry.Rejection(rejection) =>
+          stopDeduplicatingCommandSync(domain.CommandId(rejection.commandId), rejection.submitter)
```

Comment:
A rejection entry can be added:
- By `JdbcIndexer`, when consuming a `ReadService` update
- By `SqlLedger`, when the time model check fails
- By `JdbcLedgerDao`, when persisting the transaction fails with a consistency error

Reply:
@stefanobaghino-da: does the new schema require this to be added somewhere in addition?

Reply:
The sync version of this method should probably be called inside `storeRejection` so it is done in the same database transaction.

Reply:
> The sync version of this method should probably be called

This is the sync method 🙂

Reply:
> @stefanobaghino-da: does the new schema require this to be added somewhere in addition?

The new schema has a separate table for completions and rejections, I believe we can wire it in there.

Reply:
I'll try to transition this to the new schema after this has been merged.